server/app.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+const http = require('http');
+const express = require('express');
+const bodyParser = require('body-parser');
+const _ = require('lodash');
+const fs = require('fs');
+const usersRouter = require('./routes/user');
```

Comment:
use another way to call routers. We don't need to add all the files in the app.js

Reply:
Done.